specification/migrate/resource-manager/Microsoft.Migrate/stable/2019-05-01/migrate.json

```diff
@@ -0,0 +1,3748 @@
+        "2019-05-01"
+      ]
+    },
+    "acceptLanguage": {
+      "description": "Standard request header. Used by service to respond to client in appropriate language.",
+      "name": "Accept-Language",
+      "type": "string",
+      "in": "header"
+    }
+  },
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "definitions": {
+    "Project": {
+      "type": "object",
+      "x-ms-azure-resource": true,
+      "description": "Azure Migrate Project.",
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "Path reference to this project /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Migrate/assessmentProjects/{projectName}",
+          "readOnly": true
+        },
+        "name": {
+          "type": "string",
+          "description": "Name of the project.",
+          "readOnly": true
+        },
+        "type": {
+          "type": "string",
+          "description": "Type of the object = [Microsoft.Migrate/assessmentProjects].",
+          "readOnly": true
+        },
+        "eTag": {
+          "type": "string",
+          "description": "For optimistic concurrency control."
+        },
+        "location": {
+          "type": "string",
+          "description": "Azure location in which project is created."
+        },
+        "tags": {
+          "type": "object",
+          "description": "Tags provided by Azure Tagging service."
+        },
+        "properties": {
+          "description": "Properties of the project.",
+          "$ref": "#/definitions/ProjectProperties"
+        }
+      }
+    },
+    "ProjectProperties": {
+      "type": "object",
+      "description": "Properties of a project.",
+      "properties": {
+        "createdTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this project was created. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "updatedTimestamp": {
+          "type": "string",
+          "format": "date-time",
+          "description": "Time when this project was last updated. Date-Time represented in ISO-8601 format.",
+          "readOnly": true
+        },
+        "serviceEndpoint": {
+          "type": "string",
+          "description": "Endpoint at which the collector agent can call agent REST API.",
+          "readOnly": true
+        },
+        "assessmentSolutionId": {
+          "type": "string",
+          "description": "Assessment solution ARM id tracked by Microsoft.Migrate/migrateProjects."
```

Comment:
Is this readonly? Could user set this field?

Reply:
No this is not read only. User can set this field.